src/core_ocean/analysis_members/Registry_mixed_layer_heat_budget.xml

```diff
@@ -0,0 +1,107 @@
+		<nml_record name="AM_mixedLayerHeatBudget" mode="forward;analysis">
+		<nml_option name="config_AM_mixedLayerHeatBudget_enable" type="logical" default_value=".false." units="unitless"
+			description="If true, ocean analysis member mixedLayerHeatBudget is called."
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_AM_mixedLayerHeatBudget_compute_interval" type="character" default_value="output_interval" units="unitless"
+			description="Timestamp determining how often analysis member computation should be performed."
+			possible_values="Any valid time stamp, 'dt', or 'output_interval'"
+		/>
+		<nml_option name="config_AM_mixedLayerHeatBudget_output_stream" type="character" default_value="mixedLayerHeatBudgetOutput" units="unitless"
+			description="Name of the stream that the mixedLayerHeatBudget analysis member should be tied to."
+			possible_values="Any existing stream name or 'none'"
+		/>
+		<nml_option name="config_AM_mixedLayerHeatBudget_compute_on_startup" type="logical" default_value=".true." units="unitless"
+			description="Logical flag determining if an analysis member computation occurs on start-up."
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_AM_mixedLayerHeatBudget_write_on_startup" type="logical" default_value=".true." units="unitless"
+			description="Logical flag determining if an analysis member write occurs on start-up."
+			possible_values=".true. or .false."
+		/>
+	</nml_record>
+	<packages>
+		<package name="mixedLayerHeatBudgetAMPKG" description="This package includes variables required for the mixedLayerHeatBudget analysis member."/>
+	</packages>
+	<var_struct name="mixedLayerHeatBudgetAM" time_levs="1" packages="mixedLayerHeatBudgetAMPKG">
```

Comment:
The temporal tendency is not included in  the output stream? Is there something  that we want, for closure?